gitops/openshift/apps/automationtests-api/template/jag-adobe-common-service-tests.env

```diff
@@ -0,0 +1,14 @@
+APP_NAME=jag-adobe-common-service
+COMPONENT_NAME=jag-adobe-common-service-tests
+OC_NAMESPACE=c71a03
```

Comment:
to be updated when a oc namespace is created?

Reply:
Yes